modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -269,11 +261,12 @@ $admin-bar-border: 1px solid var(--a-base-9);
 }
 
 .apos-admin-bar__create {
-  margin-left: 20px;
+  margin-left: 10px;
   // Adjust button padding and svg size to have a large plus icon while keeping
   // the button size the same.
   /deep/ .apos-context-menu__btn {
-    padding: 4px;
+    padding: 2px;
+    border: 0;
```

Comment:
If I find something else, then this would be better as `border-width: 0` imo. If I don't, it's fine to leave.

Reply:
Reason being that it sets the important property directly rather than re-delcaring all the border property default values.